nautobot/docs/configuration/optional-settings.md

```diff
@@ -257,6 +257,24 @@ The global Celery task hard timeout (in seconds). Any background task that excee
 
 ---
 
+## CELERY_WORKER_PROMETHEUS_PORT
+
++++ 1.5.x
+
+Default: `` (disabled)
+
+Environment Variable: `NAUTOBOT_CELERY_WORKER_PROMETHEUS_PORT`
+
+Ports for prometheus metric HTTP server running on the celery worker.
+
```

Comment:
Nit: would it be more accurate to change the variables to plural, i.e. `CELERY_WORKER_PROMETHEUS_PORTS` and `NAUTOBOT_CELERY_WORKER_PROMETHEUS_PORTS`?

Reply:
Done.